node/aggregator.go

```diff
@@ -31,6 +31,8 @@ type aggregator struct {
 	dalc     da.DataAvailabilityLayerClient
 	executor *state.BlockExecutor
 
+	newHeaders chan *types.Header
```

Comment:
Is it idiomatic to call a channel `newHeaders` (mostly the plural is bugging me)? Why not something like `headerChan` or `newHeaderCh`?